src/components/core/DpDataTable/DpDataTable.vue

```diff
@@ -1,19 +1,19 @@
 <script>
 import { CleanHtml } from '../../../directives/CleanHtml/CleanHtml'
 import DomPurify from 'dompurify'
+import DpDraggable from '../DpDraggable'
 import DpLoading from '../../DpLoading/DpLoading'
 import DpTableHeader from './DpTableHeader'
 import DpTableRow from './DpTableRow'
-import draggable from 'vuedraggable'
 
 export default {
   name: 'DpDataTable',
 
   components: {
+    DpDraggable,
```

Comment:
If async imported, only DataTable instances that are draggable, load the additional lib. 

```
DpDraggable: () => import('../DpDraggable')
```

In the `DpTreeList` and `DpTreeListNode` components it is a little more complicated, as those use the `<dp-draggable>` element also if not draggable. We would have to work with `<component :is="" />` within those templates to be able to actually split out the draggable lib from the main bundle.

Reply:
I don't quite understand this. What should I put inside `<component :is="condition" />`? On what should it be dependent?

Reply:
If DpTreeList/DpTreeListNode is not draggable (via its `draggable` prop), it would not have to load the DpDraggable component. So we could set the component that renders the list like so: 

```
<component :is="draggable ? 'dp-draggable' : 'div' />
```

This would allow us to `import()` DpDraggable, reducing the size of the bundled demosplan-ui.

Reply:
Ah, now I understand, thanks. Done in [6b82ae8](https://github.com/demos-europe/demosplan-ui/pull/53/commits/6b82ae815e4fb8677bf54bcc3f461efbe2cd771b)

Reply:
Thank you for approaching this! I checked out the branch to also add the lazy imports `DpDraggable: () => import('../DpDraggable'),` in DpTreeList and DoTreeListNode, however this does not lead to DpDraggable being split out when building all of it. I don't know whats the problem here, so i assume further investigation is needed. However to not further block the merge of this PR, i think this may as well be tackled later.